### ISSUE TITLE
Recommended System Requirements

### ISSUE THREAD

**Issue by danzy** [wazuh/docker-ossec-elk#4]
Hi!

Are there any recommended system requirements? 

I run into a problem with the container after a day or two of having the server running (with one client connected), I wouldn't be able to load the dashboard (I'd get "Connection Refused"). 

I'm able to restart the entire container, but that's obviously not ideal to do every day. 

The container is running on an AWS t2.small instance (Intel Xeon 3.3GHz, 2GB RAM) which, given the fact that I only have one client connected to the server, should handle it?

Any help would be greatly appreciated.

Thanks!

_edit: I did have another issue included in this GitHub issue, but I realised, and fixed, my problem 5 minutes after clicking Submit. Sorry!_

**Comment by jlruizmlg** [comment]
Hi @danzy, i'm running this container for days without problem in a t2.small like you, also in a t2.micro, but anyway the Elasticsearch documentation says not less than 16GB and the optimal is 64GB.

https://www.elastic.co/guide/en/elasticsearch/guide/current/hardware.html#_memory

The "Connection Refused" is when try to load the dashboard? or when you try to log in to Kibana?

Inside the container you have in ```/var/log``` the different logs from Elasticsearch and Kibana, can you take a look and paste here if you have any error?

The next time you have this issue again,  log in the container and run this commands in order to verify the elasticsearch installation:

- ```curl -XGET localhost:9200```

- ```curl -XGET 'http://localhost:9200/_cluster/health?pretty=true'```

with this information we can start to figure out  what is your problem.

Thanks for your feedback!!

**Comment by jlruizmlg** [comment]
hi @danzy any update to this issue?

**Comment by danzy** [comment]
Hey @jlruizmlg: I'm sorry but I've stopped research on OSSEC/Kibana for the moment. 

I'm happy to close this issue (and treat it as something wrong on my side), and if I experience this issue again when I get back to it, I'll open another ticket with more information.

From memory though, I was receiving the "Connection Refused" message was appearing when I tried to log into Kibana, but I don't have any logs available to give you more information.
